treeherder/config/settings.py

```diff
@@ -114,7 +120,7 @@
 #
 # which django-environ converts into the Django DB settings dict format.
 DATABASES = {
-    'default': env.db_url('DATABASE_URL'),
+    'default': env.db_url('DATABASE_URL', default='mysql://root@localhost:3306/treeherder' if IS_WINDOWS == "windows" else 'mysql://root@127.0.0.1:3306/treeherder'),
```

Comment:
Do this a couple of lines above (check that `'%s'.format('foo')` is the right syntax; don't assume I got it right).
```python
LOCALHOST_MYSQL_HOST='mysql://root@%s:3306/treeherder'.format('localhost' if IS_WINDOWS else '127.0.0.1')
DATABASES = {
    'default': env.db_url('DATABASE_URL'), default=LOCALHOST_MYSQL_HOST,
```

`IS_WINDOWS` is a boolean and not a string.

Reply:
Okay. Was thinking of implementing this format method :). Let me correct it